pkg/cmd/opts/install.go

```diff
@@ -780,7 +780,35 @@ func (o *CommonOptions) InstallGcloud() error {
 	if runtime.GOOS != "darwin" || o.NoBrew {
```

Comment:
I think this if statement is not quite correct anymore. With the code below we can install `gcloud` for any OS into _JX_HOME/bin_, right?

Reply:
Also in the current form the use of the _NoBrew_ flag does not make sense anymore, since we are not using brew in this function at all.

Reply:
Yeah with the current code the binary gets downloaded into the `JX_HOME/bin` . So IMO there is not point of the NoBrew flag anymore.

Reply:
Thought we agreed on removing this?

Reply:
Also, I am on macOS and for me the test is not even properly executing since it returns an error here. If anything it should be `!o.NoBrew` (I know this inverted logic is tricky)

Reply:
Agreed, I might have missed it while pushing the change.